.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+name: Docs - Build and Deploy
```

Comment:
this commit should have a prefix such as `github: main:` or just `github: Add doc deployment action`

Reply:
This one is definetly badly commited.
Changing.

Reply:
Done.